app/models/user.rb

```diff
@@ -16,7 +16,7 @@ class User < ApplicationRecord
   has_many   :miq_widget_sets, :as => :owner, :dependent => :destroy
   has_many   :miq_reports, :dependent => :nullify
   has_many   :service_orders, :dependent => :nullify
-  has_many   :shares
+  has_many   :owned_shares, :class_name => "Share"
```

Comment:
Promise me we are not going to route all lookups through `user.owned_shares`. e.g:

- `user.owned_shares.grep(DesiredClass)`
- `MiqPreloader.preload(user.owned_shares)`

Lets just keep those bad code examples in mind as we move forward